docs_src/microservices/device/how-to/Ch-RemoteSecure.md

```diff
@@ -4,4 +4,420 @@ title: Device Service - Remote deployment secure mode
 
 # Device Service - Remote deployment secure mode
 
-Coming Soon.
+In some use cases, devices are connected to nodes where core EdgeX services are not running. In these cases the appropriate device service
+needs to run on the remote nodes where it can connect to the device(s) and communicate to the host node where the rest of the EdgeX services are running.
+
+This page provides an example of configuring and deploying multiple instances of [device-usb-camera](../services/device-usb-camera/General.md)
+service using multiple nodes in EdgeX secure mode. This example is applicable to any device service.
+
+## Example
+This example uses 3 nodes for remote deployment. One of the nodes (host) is used to run all EdgeX core services in Docker and the other nodes (remote)
+for running the device-usb-camera service in Docker. This example can be further extended to run multiple instances of device-usb-camera
```

Comment:
per " This example can be further extended to run multiple instances of device-usb-camera
service in multiple nodes." 

Are you do that in this example with 2 remote nods each running Device USB Camera?